### DISCUSSION TITLE
Azure Web App update Wiki JS version

### DISCUSSION THREAD

**Discussion by SafeBV** [requarks/wiki#4952] [Help / Questions]
Hi,

We have Wiki JS running on Azure Database for PostgreSQL server and web app. We are relatively new with azure, and we would like to upgrade Wiki JS to the new version. But we don't know how to get to the new version. Is there some manual or so for this?

Thnx in advanced!
![wikijs](https://user-images.githubusercontent.com/98760063/151823268-27215ab9-6712-42f6-952e-a369be3856e2.png)

**Comment by Phil-hostwiki** [comment]
Login to your Azure machine via  SSH and follow  the docker upgrade instruction on this page https://docs.requarks.io/install/upgrade.

**Comment by gitwrckd** [comment]
That's not accurate for a web app.  That would be for an actual deployed VM with docker etc.

In this case... as simple as it sounds, hit restart on your web app.  It works; I just did it as I had the same question.

![image](https://user-images.githubusercontent.com/99357479/153270453-939866d8-eb12-44ec-8bb0-2430743b5cb7.png)


See point 4: https://docs.microsoft.com/en-us/azure/developer/python/tutorial-deploy-containers-03
